apps/sr-frontend/src/app/Character.tsx

```diff
@@ -55,6 +60,114 @@ export default function Character() {
                 })
               }
             />
+            <Grid container>
```

Comment:
We should disable all buttons (except the characterselector) if the character object is empty

Reply:
Should dropdowns also be disabled or only buttons?